src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -132,7 +141,8 @@ const ChatItemMyTextV2 = ({
             previewBackgroundColor="transparent"
             style={{flex: 1}}
             actions={contextMenuActions}
-            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}
+            disabled={attachments.length > 1}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPress` event now checks if there are more than one attachments before triggering. This is a good practice as it prevents unnecessary function calls when there are no attachments.



<!-- This is an auto-generated comment by OSS CodeRabbit -->